File: 1561-maximum-number-of-coins-you-can-get/1561-maximum-number-of-coins-you-can-get.ts
function maxCoins(piles: number[]): number {
    let me = 0;
        piles.sort((a,b)=>a-b);
        for (let j = piles.length - 2, i = 0; i < j; i++, j -= 2)
            me += piles[j];
        return me;
};

File: 1561-maximum-number-of-coins-you-can-get/1561-maximum-number-of-coins-you-can-get.py
class Solution:
    def maxCoins(self, piles: List[int]) -> int:
        me = 0
        piles.sort()
        
        j = len(piles)-2
        i = 0
        
        while i < j:
            me += piles[j]
            i+=1
            j -= 2
        
        return me